src/server.py

```diff
@@ -13,6 +24,8 @@
 @app.route('/topimage',methods = ['GET'])
 def api_top_image():
     url = request.args.get('url')
+    if is_content_pdf(url):
+        return EMPTY_PDF_RESPONSE
     article = get_article(url)
```

Comment:
In case we are looking for more elegant way of implementation, Article that has 
`def is_valid_url(self):` which can be overridden to serve this purpose.